source/org.tensorflow/tensorflow-lite-task-audio-play-services/Transforms/Metadata.Namespaces.xml

```diff
@@ -1,10 +1,10 @@
 ﻿<metadata>
-    <!--
     <attr 
         path="/api/package[@name='package']" 
```

Comment:
This XPath looks incorrect.

Reply:
I think I messed up merge. Let me fix it. It will be ready tomorrow.